app/views/subjects/_form.html.haml

```diff
@@ -0,0 +1,16 @@
+= form_for @subject do |f|
+  .field
+    = f.label :name
+    = f.text_field :name
+  .field
+    = f.label :desc
+    = f.text_area :desc
+  .actions
+    = f.submit
+  - if @subject.errors.any?
+    #error_explaination
+    %h2= pluralize(@subject.errors.count, 'error')
+    prohibited this subject from being saved:
```

Comment:
I will change to what you had suggested in a previous comment `%h2 #{pluralize @subject.errors.count, 'error'} prohibited this subject from being saved:`